deploy/cluster_role.yaml

```diff
@@ -45,6 +45,8 @@ rules:
   - accountclaims
   - accounts
   - accountpools
+  - awsfederatedaccountaccesses
```

Comment:
Do we want to keep the * on Line 44?

Reply:
I think we chose to keep it last time but be also be specific, I'm happy to remove it but in a separate PR in case we break something.